crates/agent-sql/src/publications.rs

```diff
@@ -540,7 +540,7 @@ pub async fn resolve_storage_mappings(
         where starts_with(n, m.catalog_prefix)
            or starts_with('recovery/' || n, m.catalog_prefix)
            -- TODO(johnny): hack until we better-integrate ops collections.
-           or m.catalog_prefix = 'ops/'
+           or m.catalog_prefix = 'ops.us-central1-c.v1/'
```

Comment:
nit: think there was a crossed wire here -- per the slack thread, we should include the region but not the availability zone